utils.py

```diff
@@ -357,7 +357,9 @@ def get_os():
 
 def get_package_manager():
     # Check for superuser command
-    if shutil.which('sudo') is not None:
+    if shutil.which('pkexec') is not None:
+        config.SUPERUSER_COMMAND = "pkexec"
+    elif shutil.which('sudo') is not None:
         config.SUPERUSER_COMMAND = "sudo"
     elif shutil.which('doas') is not None:
         config.SUPERUSER_COMMAND = "doas"
```

Comment:
When I implemented more supported dialogs in Bash, I had a simple flag to check whether we were on GTK or Plasma; the flag allowed me to switch the order of the requested dialog so that GTK would get zenity and Plasma would get kdialog. Though kdialog was never implemented, the change would have made a big difference for making the installer feel integrated.

I think we should add a flag here to check if we are TK or TUI/CLI and swap the order, and that way in TK we don't even request sudo or doas since they would never appear anyways.